src/refract.c

```diff
@@ -78,7 +78,7 @@ double novas_inv_refract(RefractionModel model, double jd_tt, const on_surface *
 /**
  * Returns an optical refraction correction for a standard atmosphere.
  *
- * @param jd_tt     [day] Terrestrial Time (TT) based Julian data of observation
+ * @param jd_tt     [day] Terrestrial Time (TT) based Julian data of observation (currently unused)
```

Comment:
Rather than "(currently unused)" I's say "(unused in this implementation of RefractionModel)"